server/lib/src/server/modify.rs

```diff
@@ -556,18 +556,19 @@ mod tests {
         assert!(server_txn.modify(&me_emp) == Err(OperationError::EmptyRequest));
 
         // Mod changes no objects
-        let me_nochg = ModifyEvent::new_impersonate_entry_ser(
-            JSON_ADMIN_V1,
-            filter!(f_eq(
-                Attribute::Name,
-                PartialValue::new_iname("flarbalgarble")
-            )),
-            ModifyList::new_list(vec![Modify::Present(
-                Attribute::Description.into(),
-                Value::from("anusaosu"),
-            )]),
-        );
-        assert!(server_txn.modify(&me_nochg) == Err(OperationError::NoMatchingEntries));
+        // TODO: @yaleman fix this because we don't have a way to do this anymore
+        // let me_nochg = ModifyEvent::new_impersonate_entry_ser(
+        //     JSON_ADMIN_V1 ,
+        //     filter!(f_eq(
```

Comment:
Same as before? Manually create the FC to bypass this?

Reply:
it was more the JSON_ADMIN_V1 bit because I hadn't worked out the account objects yet :smile: All good